(DOCSP-44990) Updating High Availability Page (#95)
mongodb/docs-atlas-architecture: source/high-availability.txt

Recommended Deployment Topologies
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Single Region, 3 Node Replica Set / Shard (Primary with Two Secondary Members)

Reviewer: Think this format might be more clear for the other ones
```suggestion
Single Region, 3 Node Replica Set / Shard (Primary Secondary Secondary)
```

````````````````````````````````````````````````
This topology is appropriate if low latency is required but high availability requirements 

> are limited to a single region. This topology can tolerate any 1 node failure, easily satisfy majority write 

Reviewer: ```suggestion
are limited to a single region. This topology can tolerate any 1 node failure and easily satisfy majority write 
```

concern within region secondaries. This will maintain a primary in the preferred region upon any node 
failure, limits cost, and is the least complicated from an application architecture perspective. 
The reads and writes from secondaries in this topology encounter low latency in your preferred region.

This topology, however, can't tolerate a regional outage.

> 3-Region, 3 Node Replica Set / Shard (P - S - S)

Reviewer: ```suggestion
3-Region, 3 Node Replica Set / Shard (Primary - Secondary - Secondary)
```

``````````````````````````````````````````````````
This topology is the standard multi-regional topology where high availability can be provided to tolerate a regional outage. 
This topology can tolerate any 1 node failure, any 1 region outage, and is the least expensive multi-region topology. 

If the application requires high durability and the app server code has the write majority option set 
in the driver, this topology will experience higher latency for writes as they need to replicate to a second region. 

> Additionally, any reads off the secondary always exist in a different region than the preferred primary region which 

Reviewer: ```suggestion
Additionally, any reads off the secondary always exist in a different region than the preferred primary region
```

> and will require a different app server architecture. Furthermore, any failover event will occur to a different region and your 

Reviewer: ```suggestion
and will require a different app server architecture. Furthermore, any failover event will occur to a different region, and your 
```

application architecture must adjust as a result.

> 3-Region, 5 Node Replica Set / Shard (PS - SS - S)

Reviewer: ```suggestion
3-Region, 5 Node Replica Set / Shard (Primary Secondary - Secondary Secondary - Secondary)
```

````````````````````````````````````````````````````
This topology is the preferred topology that balances high availability, performance, and cost across multiple regions. 
This topology can tolerate any 2 nodes' failure, tolerate primary node failure 
while keeping the new primary in the preferred region, and tolerate any 1 region outage. 

If the application requires high durability and the app server code has the write majority option set 
in the driver, this topology will experience higher latency for writes as they need to replicate to a second region. 

> In order to accommodate a regional outage, the application tier must be configured to also failover and shift work to 

Reviewer: I believe this is two words as a verb, one as a noun
```suggestion
In order to accommodate a regional outage, the application tier must be configured to also fail over and shift work to 
```
